content/docs/rfcs/51/README.md

```diff
@@ -173,45 +169,80 @@ This example node is part of shards `13`, `14`, and `45` in the Status main-net
 
 # Automatic Sharding
 
-> *Note:* Automatic sharding is not yet part of this specification.
-This section merely serves as an outlook.
-A specification of automatic sharding will be added to this document in a future version.
+Autosharding select shards automatically and is the default behaviour for shard choice but opting out is always possible.
+Shards (pubsub topics) not longer have to be chosen, they are instead computed from content topics with the procedure below.
```

Comment:
Don't think we need to refer to history here "no longer". Perhaps something like:

```suggestion
With autosharding, shards (pubsub topics) do not have to be explicitly indicated,
but are instead computed from content topics with the procedure below.
```